src/server/csp.ts

```diff
@@ -143,9 +143,7 @@ function buildCspEditions(
 			? 'https://platform.twitter.com https://cdn.syndication.twimg.com'
 			: ''
 	};
-	frame-src https://www.theguardian.com ${
-		thirdPartyEmbed.instagram ? 'https://www.instagram.com' : ''
-	} https://www.facebook.com https://www.tiktok.com https://interactive.guim.co.uk ${
+	frame-src https://www.facebook.com https://www.tiktok.com https://interactive.guim.co.uk ${
```

Comment:
Did you mean to drop `www.theguardian.com` from the list of iframeable domains?

Reply:
Ah no I didn't, thanks for flagging -  added back in!